src/manage_sql.c

```diff
@@ -17595,10 +17613,14 @@ add_permissions_on_globals (const gchar *role_uuid)
 
 /**
  * @brief Ensure the predefined permissions exists.
+ *
+ * @param[in]  force_add   Whether to always try to create permissions.
  */
 static void
-check_db_permissions ()
+check_db_permissions (gboolean force_add)
```

Comment:
What's the point of having force_add and must_add?  There is only one caller, so they will both always be true.

Reply:
I wasn't sure if we want to always do the individual checks, but I've removed the conditions now.